addons/website/static/tests/tours/page_manager.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
+
+import wTourUtils from 'website.tour_utils';
+
+const delete_selected_page = [
+    {
+        content: "Pick Action",
```

Comment:
Nitpicking (don't change it, it will be a waste of time) but generally `Click on`, `Pick` seems weird